jobserver/models/core.py

```diff
@@ -877,6 +879,22 @@ def clean(self):
         super().clean()
         self.email = self.__class__.objects.normalize_email(self.email)
 
+    @classmethod
+    @transaction.atomic()
+    def from_login_token(
+        cls,
+        email,
+        signed_token,
+        max_age=timedelta(minutes=settings.LOGIN_URL_TIMEOUT_MINUTES),
+    ):
+        token = TimestampSigner(salt="login").unsign(signed_token, max_age=max_age)
```

Comment:
Worth a comment explaining that we're verifying the token in the email is valid and hasn't expired?